en_US/data-integration/data-bridge-rocketmq.md

```diff
@@ -0,0 +1,184 @@
+v# RocketMQ
+
+EMQX supports integration with RocketMQ, so you can forward client messages and events to RocketMQ, for example, use events to trigger the update of data to record the online status or online/offline of clients.
+
+{% emqxce %}
+:::tip
+EMQX Enterprise Edition features. EMQX Enterprise Edition provides comprehensive coverage of key business scenarios, rich data integration, product-level reliability, and 24/7 global technical support. Experience the benefits of this [enterprise-ready MQTT messaging platform](https://www.emqx.com/en/try?product=enterprise) today.
+:::
+{% endemqxce %}
+
+
+## Prerequisites
+
+- Knowledge about EMQX data integration [rules](./rules.md)
+- Knowledge about [data bridge](./data-bridges.md)
+
+## Features list
+
+- [Connection pool](./data-bridges.md#Connection pool)
+- [Async mode](./data-bridges.md#Async mode)
+- [Batch mode](./data-bridges.md#Batch mode)
+- [Buffer queue](./data-bridges.md#Buffer queue)
+- [SQL preprocessing](./data-bridges.md#Prepared statement)
+
+## [Configuration parameters](#Configuration)
+<!-- TODO LIKN TO THE CONFIG doc。 -->
+
+## Quick starts
+
+### Install RocketMQ 
+
+We need a docker compose file, `rocketmq.yaml`, to setup the RocketMQ
+```json
+version: '3.9'
+
+services:
+  mqnamesrv:
+    image: apache/rocketmq:4.9.4
+    container_name: rocketmq_namesrv
+    ports:
+      - 9876:9876
+    volumes:
+      - ./rocketmq/logs:/opt/logs
+      - ./rocketmq/store:/opt/store
+    command: ./mqnamesrv
+    networks:
+      - emqx_bridge
+
+  mqbroker:
+    image: apache/rocketmq:4.9.4
+    container_name: rocketmq_broker
+    ports:
+      - 10909:10909
+      - 10911:10911
+    volumes:
+      - ./rocketmq/logs:/opt/logs
+      - ./rocketmq/store:/opt/store
+      - ./rocketmq/conf/broker.conf:/etc/rocketmq/broker.conf
+    environment:
+        NAMESRV_ADDR: "rocketmq_namesrv:9876"
+        JAVA_OPTS: " -Duser.home=/opt"
+        JAVA_OPT_EXT: "-server -Xms1024m -Xmx1024m -Xmn1024m"
+    command: ./mqbroker -c /etc/rocketmq/broker.conf
+    depends_on:
+      - mqnamesrv
+    networks:
+      - emqx_bridge
```

Comment:
nit: It doesn't work for users, because the `emqx_bridge` is not defined in this code block.